main_json.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+
+def write_order_to_json(item, quantity, price, buyer, date):
+
+    with open('orders_1.json', 'r', encoding='utf-8') as f_out:

```

Comment:
It looks like the function doesn't create a json file:
```
Traceback (most recent call last):
  File "C:/Projects/Python/Client-server-applications-in-python/main_json.py", line 16, in <module>
    write_order_to_json('printer', '10', '6700', 'Ivanov I.I.', '24.09.2017')
  File "C:/Projects/Python/Client-server-applications-in-python/main_json.py", line 6, in write_order_to_json
    with open('orders_1.json', 'r', encoding='utf-8') as f_out:
FileNotFoundError: [Errno 2] No such file or directory: 'orders_1.json'

Reply:
I fixed it